apps/web/src/pages/voting/proposal/[id].tsx

```diff
@@ -26,20 +18,24 @@ export const getStaticPaths: GetStaticPaths = () => {
 }
 
 ProposalPage.Meta = (props: InferGetStaticPropsType<typeof getStaticProps>) => {
-  if (props.id && props.fallback[unstable_serialize(['proposal', props.id])]) {
-    const proposal = props.fallback[unstable_serialize(['proposal', props.id])]
-    return (
-      <NextSeo
-        openGraph={{
-          description: proposal.title,
-        }}
-      />
-    )
+  if (props.id && props.dehydratedState?.queries?.[0]?.state?.data) {
+    // @ts-ignore
+    const { title } = props.dehydratedState?.queries?.[0]?.state?.data
```

Comment:
I think it's not safe to use object destruct if the value is possibly undefined?
```suggestion
    const title = props.dehydratedState?.queries?.[0]?.state?.data?.ttile
```